pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -63,7 +63,10 @@ type Manager interface {
 	// AddContainer is called between container create and container start
 	// so that initial CPU affinity settings can be written through to the
 	// container runtime before the first process begins to execute.
-	AddContainer(p *v1.Pod, c *v1.Container, containerID string) error
+	AddContainer(p *v1.Pod, c *v1.Container, containerID string)
+
+	// GetAllocatedResources return the CPU affinity for the container
+	GetAllocatedResources(pod *v1.Pod, container *v1.Container) string
```

Comment:
I just wanted to have an interface method with the neutral name because I will want to have the same for the memory manager, WDT?

Reply:
I still think we should just consolidate it down to a single call of:
```
GetCPUs(podUID, containerName string) cpuset.CPUSet
```

We should then expand `cpuset.CPUSet` to include helper functions for:
```
ToSliceInt64() []int64
ToSliceNoSortInt64() []int64
```

Then the two call sites that use it can each call functions directly attached to the `CPUSet` type to get the format that they want.

This should be added as a standalone commit before the rest of the changes here.

Reply:
I'm happy to rename the function to `GetAllocatedCPUs()` if you like, but I think it's better to keep it CPU specific.

Reply:
Do you want a separate PR for it or the separate commit will be good enough for it?

Reply:
Separate commit is fine.